src/ast/query.rs

```diff
@@ -64,6 +64,23 @@ impl fmt::Display for Query {
     }
 }
 
+/// Options for SetOperator
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+pub enum SetOperatorOption {
+    All,
```

Comment:
In general it should be possible to round trip structures in sqlparser -- so it should be possible to distinguish between `ALL`, `DISTINCT` or nothing was specified in the query

Reply:
I'll add the `None` to the enum `SetOperatorOption`, and remove `Option<T>` for `SetOperatorOption`.

Reply:
I fixed this at [ac41a8e](https://github.com/sqlparser-rs/sqlparser-rs/pull/689/commits/ac41a8e839e1a058c9957d6676ffa269cfd82366)